packages/payment-integration/coverage/base.css

```diff
@@ -0,0 +1,224 @@
+body, html {
```

Comment:
Shall we add this in .gitignore? Feels like something we don't need to commit.

Reply:
Yes, I didn't realise it is committed. Thanks for spotting this!